backend/ops_api/ops/resources/can_funding_summary.py

```diff
@@ -13,8 +13,53 @@ def __init__(self, model: BaseModel):
         super().__init__(model)
 
     @is_authorized(PermissionType.GET, Permission.CAN)
-    def get(self, id: int) -> Response:
+    def get(self) -> Response:
+        # Get query parameters
+        can_ids = request.args.getlist("can_ids")
         fiscal_year = request.args.get("fiscal_year")
-        can = self._get_item(id)
+        active_period = request.args.getlist("active_period", type=int)
+        transfer = request.args.getlist("transfer")
+        portfolio = request.args.getlist("portfolio")
+        fy_budget = request.args.getlist("fy_budget", type=int)
+
+        # Check if required 'can_ids' parameter is provided
+        if not can_ids:
+            return make_response_with_headers({"error": "'can_ids' parameter is required"}, 400)
```

Comment:
The above `request.args` handling and `can_ids` validation would be better in a Marshmallow schema but it would be acceptable to take that as a tech debt item if you don't have time to implement it.